src/backend/marsha/bbb/utils/bbb_utils.py

```diff
@@ -179,11 +192,92 @@ def join(classroom: Classroom, consumer_site_user_id, fullname, moderator=False)
     return request_api("join", parameters, prepare=True)
 
 
+def start_session(classroom: Classroom):
+    """Create a session for a given classroom,
+    and store cookie and learning analytics url."""
+    join_response = join(
+        classroom=classroom,
+        consumer_site_user_id="marsha",
+        fullname="Marsha",
```

Comment:
Does this user appears in the attendees ?

Reply:
No, as no js is executed, BBB doesn't assumes that the user joined, I suppose.
But I see no trace of this user in the classroom, or in the learning analytics.